packages/catlog/src/tt/prelude.rs

```diff
@@ -20,3 +18,7 @@ pub type FieldName = NameSegment;
 pub(super) fn text_seg(s: impl Into<Ustr>) -> NameSegment {
     NameSegment::Text(s.into())
 }
+
+pub(super) fn label_seg(s: impl Into<Ustr>) -> LabelSegment {
+    LabelSegment::Text(s.into())
+}
```

Comment:
These helper functions should be moved to the module `crate::zero::qualified`, where there are already analogous helpers `name` and `label` to construct `QualifiedName`s and `QualifiedLabel`s. Also, for consistency, `text_seg` should be renamed to `name_seg`.